### DISCUSSION TITLE
Virtual endpoint (or group)

### DISCUSSION THREAD

**Discussion by mundschenk-at** [Koenkk/zigbee2mqtt#24240] [Devices]
Is it possible to provide a virtual endpoint or group in a converter? I'd like to make Philips Hue lights with more than one endpoint more similar to how the native Hue app works (i.e. the light can be controlled "whole" or via the separate endpoints), without manually adding a ZigBee group.

**Comment by Koenkk** [comment]
Not from a converter, why not add a group?

**Reply by mundschenk-at** [reply]
Manually adding a group works fine in principle, but it has two drawbacks:

1. The resulting entity is not linked to the device in Home Assistant.
2. It does not work out of the box, the user has to manually create the group.
